doc/chips/11.rst

```diff
@@ -151,18 +143,19 @@ The following example relies on Syntax 2:
      }
    }
 
-Another approach is to introduce "lastinit" methods.
-One "lastinit" method with 0 arguments could be optionally defined
-on each class.
+..
+ Another approach is to introduce "lastinit" methods.
+ One "lastinit" method with 0 arguments could be optionally defined
+ on each class.
```

Comment:
I don't recall us talking about this method at all, is this just something you mused about on your own?  I'm fine with leaving that in (hidden note form) so long as you mention it wasn't brought up with the team - I don't think it's fair to put the names of the rest of us on something that we didn't talk about.